src/vector/platform/ElectronPlatform.js

```diff
@@ -66,12 +67,104 @@ function getUpdateCheckStatus(status) {
     }
 }
 
+class SeshatIndexerManager extends BaseEventIndexManager {
```

Comment:
Seems preferable to use the same suffix as the base class:

```suggestion
class SeshatIndexManager extends BaseEventIndexManager {
```